### PR TITLE
[v2, dev] Build frontend only before starting the dev watcher command

### PR DESCRIPTION
**Pull request by stffabi**
Breaking Change: Ignore flags take precedence over Force flag.

Potential fix for #1680

### CODE REVIEW

File: v2/cmd/wails/internal/commands/dev/dev.go [hunk 1]
```diff
@@ -473,19 +467,25 @@ func runFrontendDevWatcherCommand(cwd string, devCommand string, discoverViteSer
 	LogGreen("Running frontend DevWatcher command: '%s'", devCommand)
 	var wg sync.WaitGroup
 	wg.Add(1)
+	var state int32
 	go func() {
 		if err := cmd.Wait(); err != nil {
-			if err.Error() != "exit status 1" {
+			wasNotCanceled := atomic.CompareAndSwapInt32(&state, 0, 2)
```
Comment (leaanthony) [inline v2/cmd/wails/internal/commands/dev/dev.go:473]: Was wondering if we should define consts for the 0,1 and 2 values for readability?

**Review by leaanthony — APPROVED** [review]

**Comment by akhudek** [comment]
This PR worked for my problem in https://github.com/wailsapp/wails/issues/1680

File: v2/cmd/wails/internal/commands/dev/dev.go (hunk 1 above)
Comment (stffabi) [reply v2/cmd/wails/internal/commands/dev/dev.go]: Yeah, absolutely 👍

**Comment by leaanthony** [comment]
Awesome stuff 🚀🚀🚀

**Review by leaanthony — APPROVED** [review]

**Comment by leaanthony** [comment]
Nice one. I'll look at doing a release tomorrow 👍

**Comment by stffabi** [comment]
Awesome, thanks for merging.

**Pull request merged**